src/mixins/nr-affiliation-mixin.ts

```diff
@@ -141,7 +145,7 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
     let businessRequest = {} as BusinessRequest
     const nrNumber = nr.nrNum
     if (!this.isFirm(nr)) {
```

Comment:
I recommend you refactor this by moving the firm case first and the others into the else. That keeps the conditional statement and its code block close together for easier understanding.

Alternately, you could leave this line alone and change line 186 to `if (this.isFirm(nr))`.

Reply:
Alternately you could leave this alone 😆

Reply:
Added the change to line 186.

Reply:
Yeah, this is cleaner, given the number of if-then-else and curly brackets 👍